package.json

```diff
@@ -10,9 +10,12 @@
     "@types/node": "^16.11.18",
     "@types/react": "^17.0.38",
     "@types/react-dom": "^17.0.11",
+    "dotenv": "^10.0.0",
```

Comment:
create-react-app už má envy naprogramovaný 
https://create-react-app.dev/docs/adding-custom-environment-variables/

tak je to podle mě zbytečný přidávat ne?